plugins/transport-reactor-netty4/src/javaRestTest/java/org/opensearch/rest/ReactorNetty4StreamingIT.java

```diff
@@ -304,7 +304,34 @@ public void testStreamingLargeDocument() throws IOException {
             String.format(
                 Locale.getDefault(),
                 "{ \"index\": { \"_index\": \"test-streaming\", \"_id\": \"1\" } }\n{ \"name\": \"%s\"  }\n",
-                randomAlphaOfLength(5000)
+                randomAlphaOfLength(7000)
+            )
+        );
+
+        final Duration delay = Duration.ofMillis(1);
```

Comment:
I don't see this variable being used anywhere?

Reply:
Ah, leftovers, thanks @andrross , removing